transmogrifier/models.py

```diff
@@ -43,11 +49,26 @@ def not_empty(
         raise ValueError(message)
 
 
+@define
+class ListField:
+    def __hash__(self) -> int:
+        """Hash method to create unique identifier for Location objects."""
```

Comment:
Minor: think it's more generic than just `Location objects`?

Reply:
Ah, typo! 😅 Will update!

Reply:
Added a standalone `timdex_object_hash` function with docstring!